gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/computer/bulkloading/BulkLoaderVertexProgram.java

```diff
@@ -325,9 +325,12 @@ private Builder() {
 
         @SuppressWarnings("unchecked")
         @Override
-        public BulkLoaderVertexProgram create(final Graph graph) {
-            ConfigurationUtils.append(graph.configuration().subset(BULK_LOADER_VERTEX_PROGRAM_CFG_PREFIX), configuration);
-            return (BulkLoaderVertexProgram) VertexProgram.createVertexProgram(graph, configuration);
+        public BulkLoaderVertexProgram create(final Graph... graphs) {
+            if (graphs.length == 0) {
+                throw new IllegalStateException("Must provide at least one graph to use");
```

Comment:
Better to use a `IllegalArgumentException` in this case as the validation is on the an argument to `create()` ? I think it should be changed in all the "programs" as they all seem to validate in the same manner.

Reply:
Done.